project/GenTupleAggregators.scala

```diff
@@ -66,8 +66,17 @@ object MapAggregator {
   }
 
   def genMapMethods(max: Int, isMonoid: Boolean = false): String = {
+    val aggType = if (isMonoid) "Monoid" else ""
+
+    // there's no Semigroup[Tuple1[T]], so just use T as intermediary type instead of Tuple1[T]
+    val aggregatorForOneItem = s"""
+       |def apply[A, B, C](aggDef: (String, ${aggType}Aggregator[A, B, C])): ${aggType}Aggregator[A, B, Map[String, C]] = {
```

Comment:
can this be `(K, ${aggType}Aggregator[A, B, C])` so it matches the others? We should not require a String as the key, I don't think.